File: Python/do_they_match.py
def do_they_match(str, ending):
    print(slice(len(str) - len(ending), len(str) - 1, 1))
    return slice(len(str) - len(ending), len(str) - 1, 1) == ending


print(do_they_match("Ernesto", "to"))  # true

File: JavaScript/doTheyMatch.js
function doTheyMatch(str, ending) {
  return str.slice(str.length - ending.length) === ending;
}

console.log(doTheyMatch('Ernesto', 'to')); // true
console.log(doTheyMatch('Xaia', 'd')); // false
console.log(doTheyMatch('Linzi', 'nzi')); // true

function sliceItUp(string, low, high) {
  return string.slice(low, high);
}

console.log(sliceItUp('Orlando', 2, 6)); // 'land'
